.github/workflows/benchmarks-integration-tests.yml

```diff
@@ -77,14 +76,49 @@ jobs:
             echo "candidate-ref=${{ env.ERA_COMPILER_LLVM_REF_DEFAULT }}" | tee -a "${GITHUB_OUTPUT}"
           fi
 
+  target-machine:
+    runs-on: ubuntu-latest
+    outputs:
+      evm: ${{ steps.evm.outputs.machine }}
+      eravm: ${{ steps.eravm.outputs.machine }}
+      evmemulator: ${{ steps.eravm.outputs.machine }}
+      default: ${{ steps.default.outputs.machine }}
+    steps:
+
+      - name: Check for EraVM target
+        id: eravm
+        if: contains(github.event.pull_request.title, '[EraVM]')
+        run: echo "machine=EraVM" | tee -a "${GITHUB_OUTPUT}"
+
+      - name: Check for EVM target
+        id: evm
+        if: contains(github.event.pull_request.title, '[EVM]')
+        run: echo "machine=EVM" | tee -a "${GITHUB_OUTPUT}"
+
+      - name: Check for  target
+        id: evmemulator
+        if: contains(github.event.pull_request.title, '[EVMEmulator]')
```

Comment:
I'd remove `EVMEmulator` here for now.
We'll probably have another flag to select the environment. LLVM should only care about the instruction set itself.

Reply:
Sure, removed.